core/src/lib.rs

```diff
@@ -392,12 +384,7 @@ pub unsafe trait Field {
         ptr.cast::<u8>().sub(self.field_offset()).cast()
     }
 
-    /// projects the raw pointer from the field `Type` to the `Parent` type
-    ///
-    /// # Safety
-    ///
-    /// * `ptr` must point into a valid allocation of `Type`
-    /// * `ptr` must point to a field of `Parent` with the type `Type`
+    /// Project a raw pointer from a `Type` to its generating `Parent`
```

Comment:
The safety comments here are important.  They should remain in the docs even if they are repeated elsewhere (people tend to skim docs and may miss the important bits unless they are repeated).

Reply:
nice catch!